packages/cozy-harvest-lib/src/components/OAuthWindow.jsx

```diff
@@ -195,6 +203,8 @@ OAuthWindow.propTypes = {
   redirectSlug: PropTypes.string,
   /** Is it a reconnection or not */
   reconnect: PropTypes.bool,
+  /** Is it a manage or not */
+  manage: PropTypes.bool,
   /** Existing account */
   account: PropTypes.object,
   /** custom intents api. Can have fetchSessionCode, showInAppBrowser, closeInAppBrowser at the moment */
```

Comment:
I think that we should have a  "lower" `OAuthWindow` component. And create a new `BIOAuthWindow` for instance because if I need to develop an OAuth Konnector and then I want to "edit" the account, reading "Is it a manage or not" I will set it to `true`... 

But manage & reconnect are very specific to BI not standard OAuth. Same for `getOauthData`

Reply:
Reconnect is actually not specific to BI and could be used for the Google connector for example to renew an OAuth token when refresh token is expired. 
And from the stack point of view, the manage route could also be used outside of BI (but yes, I don't have a real use case here for the moment). That is why I did not mention BI in this component.

I will try to make my descriptions better